packages/main/src/TimePicker.js

```diff
@@ -0,0 +1,723 @@
+				style: this._formatPattern,
+			});
+		}
+
+		return this._oDateFormat;
+	}
+
+	setValue(value) {
+		if (this.isValid(value)) {
+			this.value = this.normalizeValue(value);
+			this.setSlidersValue();
+			this.valueState = ValueState.None;
+		} else {
+			this.valueState = ValueState.Error;
+		}
+	}
+
+	_getSlidersContained() {
+		const formatArray = this.getFormat().aFormatArray,
+			slidersBuildArray = [false, false, false, false]; // hours minutes seconds am/pm
+
+		for (let i = 0; i < formatArray.length; i++) {
+			if (this._hoursParameters.maxHour !== 0) {
+				slidersBuildArray[0] = true;
+			}
+			if (this._hoursParameters.maxHour !== 0 && this._hoursParameters.isTwelveHoursFormat) {
+				slidersBuildArray[3] = true;
+				slidersBuildArray[0] = true;
+			}
+			if (formatArray[i].type === "minute") {
+				slidersBuildArray[1] = true;
+			}
+			if (formatArray[i].type === "second") {
+				slidersBuildArray[2] = true;
+			}
+		}
+
+		return slidersBuildArray;
+	}
+
+	_initHoursFormatParameters() {
+		const formatArray = this.getFormat().aFormatArray;
+
+		if (formatArray[0].type === "hour0_23") {
+			this._hoursParameters.minHour = 0;
+			this._hoursParameters.maxHour = 23;
+			this._hoursParameters.isTwelveHoursFormat = false;
+		} else if (formatArray[0].type === "hour1_24") {
+			this._hoursParameters.minHour = 1;
+			this._hoursParameters.maxHour = 24;
+			this._hoursParameters.isTwelveHoursFormat = false;
+		} else if (formatArray[0].type === "hour0_11") {
+			this._hoursParameters.minHour = 0;
+			this._hoursParameters.maxHour = 11;
+			this._hoursParameters.isTwelveHoursFormat = true;
+		} else if (formatArray[0].type === "hour1_12") {
+			this._hoursParameters.minHour = 1;
+			this._hoursParameters.maxHour = 12;
+			this._hoursParameters.isTwelveHoursFormat = true;
+		}
+	}
+
+	/**
+	 * Currently selected date represented as JavaScript Date instance
+	 *
+	 * @readonly
+	 * @type { Date }
+	 * @public
+	 */
+	get dateValue() {
+		return this.getFormat().parse(this.value);
+	}
+
+	get shouldBuildHoursSlider() {
+		return this._getSlidersContained()[0];
+	}
+
+	get shouldBuildMinutesSlider() {
+		return this._getSlidersContained()[1];
+	}
+
+	get shouldBuildSecondsSlider() {
+		return this._getSlidersContained()[2];
+	}
+
+	get shouldBuildPeriodsSlider() {
+		return this._getSlidersContained()[3];
+	}
+
+	get hoursSliderTitle() {
+		return this.i18nBundle.getText(TIMEPICKER_HOURS_LABEL);
+	}
+
+	get minutesSliderTitle() {
+		return this.i18nBundle.getText(TIMEPICKER_MINUTES_LABEL);
+	}
+
+	get secondsSliderTitle() {
+		return this.i18nBundle.getText(TIMEPICKER_SECONDS_LABEL);
+	}
+
+	get periodSliderTitle() {
+		return this.i18nBundle.getText(TIMEPICKER_PERIODS_LABEL);
+	}
+
+	get submitButtonLabel() {
+		return this.i18nBundle.getText(TIMEPICKER_SUBMIT_BUTTON);
+	}
+
+	get cancelButtonLabel() {
+		return this.i18nBundle.getText(TIMEPICKER_CANCEL_BUTTON);
+	}
+
+	get containerClasses() {
```

Comment:
We have convention for "classes" and "styles" getters. you can look at the other components for reference,
but in general:
```js
get classes() {

  return {
          container: {
                      "ui5-timepicker-sliders-container": true,
                      "ui5-phone": isPhone(),
           }
    }
}
```

Then, the usage in the hbs is: class="{{classes.container}}"
Change it here and in the WheelSlider.js.